cmd/operator-opamp-bridge/agent/agent_test.go

```diff
@@ -167,7 +167,7 @@ func TestAgent_onMessage(t *testing.T) {
 					},
 				},
 				status: &protobufs.RemoteConfigStatus{
-					LastRemoteConfigHash: []byte("good/testnamespace401"),
+					LastRemoteConfigHash: []byte("testnamespace/good512"),
```

Comment:
I'd consider not hardcoding these values, but instead computing them in the test body. This looks like it's going to be annoying to maintain, and at a glance it's also difficult to know what this value is actually supposed to be - I had to read through the whole test setup to figure out that `512` is the length of the config file.

Reply:
true. this has definitely caused some annoyances. I'll re-work this and push that up.